.gitmodules

```
@@ -6,4 +6,4 @@
 	url = https://github.com/harbourmasters/ZAPDTR
 [submodule "OTRExporter"]
 	path = OTRExporter
-	url = https://github.com/harbourmasters/OTRExporter/
+	url = https://github.com/harbourmasters/OTRExporter
```

probably don't want this whatever it was, unless we accidentally added this in `develop-rando` earlier?
```suggestion
	url = https://github.com/harbourmasters/OTRExporter/
```

---

I have no idea, that change just came in during merge conflict resolution. The way it is now matches the way ZAPDTR's module url is formatted though, and doesn't seem to have resulted in the submodule itself being different. I can revert it if you really want but functionally there's no difference.

---

Also, the way it is in this PR matches `develop` branch.